base/sysimg.jl

```diff
@@ -96,7 +96,6 @@ include("subarray.jl")
 (::Type{Vector})() = Array{Any,1}(0)
```

Comment:
I would prefer to deprecate `Vector()` as well.

Reply:
Now that we have femtocleaner would this be a less controversial change?

Reply:
See #22717